3 things have been changed.

Crispy13 opened the pull request:
1. VAE branch output will be included model output. (VAE branch will be trained)
2. Minor change in dice score calculation (regardless of batch size)
3. SpatialDropout3D will be used instead of Dropout.

Crispy13 committed:
3 things have been changed.

1. VAE branch output will be included model output. (VAE branch will be trained)
2. Minor change in dice score calculation (regardless of batch size)
3. SpatialDropout3D will be used instead of Dropout.

IAmSuyogJadhav commented on model.py:
```diff
@@ -107,83 +107,44 @@ def sampling(args):
+
+
+def loss_gt(e=1e-8):
+    def loss_gt_(y_true, y_pred):
```
Please copy relevant parts of documentation strings from the original file and put them here (or write your own documentation strings) to make your functions informative and readable by others.

Crispy13 replied:
I edited the doc strings part you mentioned. Please check this modifications.

IAmSuyogJadhav reviewed (changes requested):
I have put down a comment regarding documentation. Please do the requested minor changes.

IAmSuyogJadhav commented:
Please confirm if you have tested these functions to work correctly. Are the model weights changing as expected after this change?

Crispy13 committed:
3 things have been changed.

1. VAE branch output will be included model output. (VAE branch will be trained)
2. Minor change in dice score calculation (regardless of batch size).
  - The dice score varied according to batch size. The reason was the dice score was not the mean of samples of one batch but it treated the whole samples of one batch as one sample which has all voxels of the samples.
3. SpatialDropout3D will be used instead of Dropout.

Crispy13 committed:
Update model.py

- Changed some of doc strings.
- Added dice_e parameter to build_model function.

Crispy13 commented:
> Please confirm if you have tested these functions to work correctly. Are the model weights changing as expected after this change?

Yes. The VAE branch is contained in the model and now trained well.

IAmSuyogJadhav reviewed (approved):
Thank you for your contribution!
